public/index.html

```diff
@@ -1,4 +1,10 @@
 <!DOCTYPE html>
+
+<link
```

Comment:
calling declaring a stylesheet outside the HTML tag? Please put that inside `<head>`